docs/2.develop/contracts/storage.md

```diff
@@ -1,17 +1,21 @@
 ---
 id: storage
-title: Storage & Data Structures
-#sidebar_label: 💾 Storage
+title: State & Data Structures
 ---
 import {CodeBlock} from '@theme/CodeBlock'
 import {CodeTabs, Language, Github} from "@site/components/codetabs"
 
-Each contract has its own storage, which **only they can modify** but [anyone can see](../../4.tools/cli.md#near-view-state-near-view-state).
+Each contract has its own state (storage), which **only they can modify** but [anyone can see](../../4.tools/cli.md#near-view-state-near-view-state).
 
-Contracts store data as key-value pairs, but our SDK enables to use **common data types** and **structures**.
+At the higher level, the contract's state is defined by the [main class attributes](./anatomy.md#defining-the-contract),
+and accessed through them.
 
-Smart contracts [pay for their storage](#storage-cost) by locking a part of their balance (~**1 Ⓝ** per **100kb**).
+Under the hood, the contract stores all the data in a key-value storage. This however is abstracted
+from you by the SDK through [serialization](./serialization.md).
 
+:::info
+Contracts [pay for their storage](#storage-cost) by locking part of their balance. Currently it costs **~1 Ⓝ** to store **100kb**
```

Comment:
Bytes is always with capital B, otherwise it is bits. As for kilo, mega, giga, Im used to seeing those in capital letters too but don't think this is an established rule

```suggestion
Contracts [pay for their storage](#storage-cost) by locking part of their balance. Currently it costs **~1 Ⓝ** to store **100KB**
```